### PR TITLE
190 mention omission of configuration options when using traveling salesman

### PR DESCRIPTION
**Pull request by Merydian**
Add comment and replace Rundreise-Optimierung with Wegpunktoptimierung in old comment.

### CODE REVIEW

File: ORStools/help/directions_from_points_1_layer.help [hunk 1]
```diff
@@ -17,6 +17,8 @@ group features afterwards.
 
 You can optionally perform a <a href="https://en.wikipedia.org/wiki/Travelling_salesman_problem">Traveling Salesman Optimization</a> on the waypoints of each (Multi)Point feature.
 
+Enabling Traveling Salesman will erase all other advanced configuration and assume the preference to be <b>fastest</b>
```
[the remark this answers is not in the document]
Comment (Merydian) [reply ORStools/help/directions_from_points_1_layer.help:20]: Good point, the settings in the UI stay the same after execution. I suppose 'ignore' would be a better term. I copied this from the tooltip in the main plugin. This should probably be changed as well?

[the remark this answers is not in the document]
Comment (koebi) [reply ORStools/help/directions_from_points_1_layer.help:20]: Huh?
There, it also states `omitted`, not `erased`?

[the remark this answers is not in the document]
Comment (Merydian) [reply ORStools/help/directions_from_points_1_layer.help:20]: When you hover over the blue statement box :) 
image

I'll change it!

[the remark this answers is not in the document]
Comment (Merydian) [reply ORStools/help/directions_from_points_1_layer.help:20]: Done

**Comment by koebi** [comment]
We use `[]` to signal optional stuff, right? Then round brackets sound good :)

**Review by koebi — APPROVED** [review]

**Pull request merged**